bootstrap-salt.sh

```diff
@@ -5835,6 +5835,8 @@ install_freebsd_git_deps() {
         /usr/local/sbin/pkg install -y ${SALT_DEPENDENCIES} python || return 1
 
         /usr/local/sbin/pkg install -y py37-requests || return 1
+        /usr/local/sbin/pkg install -y www/py-tornado4 || return 1
```

Comment:
I think you mean `py37-tornado4` here, there's no package `www/py-tornado4`

Reply:
Imho installing it as www/py-tornado4 is a bit safer way as if they change python to py38 it will still work 

```
root@fbsd114:~ # pkg info www/py-tornado4
py37-tornado4-4.5.3
Name           : py37-tornado4
Version        : 4.5.3
Installed on   : Thu Oct  8 13:08:13 2020 PDT
Origin         : www/py-tornado4
```

```
root@kitchen:~ #  pkg install -y www/py-tornado4
Updating FreeBSD repository catalogue...
FreeBSD repository is up to date.
All repositories are up to date.
The following 1 package(s) will be affected (of 0 checked):

New packages to be INSTALLED:
	py37-tornado4: 4.5.3

Number of packages to be installed: 1

The process will require 4 MiB more space.
588 KiB to be downloaded.
[1/1] Fetching py37-tornado4-4.5.3.txz: 100%  588 KiB 602.1kB/s    00:01
Checking integrity... done (0 conflicting)
[1/1] Installing py37-tornado4-4.5.3...
[1/1] Extracting py37-tornado4-4.5.3: 100%
root@kitchen:~ #
```

Reply:
That might be the case, but we should test it first before importing into bootstrap so it's better to hardcode python versions and upgrade only after careful testing.

Reply:
"Upgrade after careful testing" :))) Are u aware that we are talking here about Freebsd which is not tested in salt even not carefully ;) Anyway, changed :) Thanks for input

Reply:
That's not true, FreeBSD is covered by tests and being tested by CI with every merge.